modules/andes-core/broker/src/main/java/org/wso2/andes/server/Broker.java

```diff
@@ -189,7 +189,7 @@ private void startAMQPListener(ApplicationRegistry config, BrokerOptions options
                 } else {
                     bindAddressForHostname = InetAddress.getByName(bindAddressFromBrokerOptions);
                 }
-                String hostName = bindAddressForHostname.getCanonicalHostName();
+                String hostName = bindAddressFromBrokerOptions;
```

Comment:
Without changing variable, can't we use the variable "bindAddressFromBrokerOptions" in line 214 and 239. Since hostname is used in line 218 let's keepthis unchanged.